conf/log4j.properties.template

```diff
@@ -1,48 +0,0 @@
-#
```

Comment:
BTW, shall we write down in migration guide too?

Reply:
Yea, I think we should mention it in the migration guide. I will open another minor PR to modify migration guide.

Reply:
Actually I just added some wordings into the migration guide. Please take a look if it is okay. Thanks @HyukjinKwon